src/examples/create_release.yml

```diff
@@ -19,7 +19,7 @@ usage:
   jobs:
     build_payment_service:
       docker:
-        - image: cimg/node:lts
```

Comment:
This change is not necessary and may be misleading: we do not need to prescribe which executor to use for jobs that do not use steps from the `pipeline-feedback` orb, or do I miss some fine point of CircleCI, @mehdicopter ? :-)

Reply:
No you are right.